### DISCUSSION TITLE
Which version is the true version

### DISCUSSION THREAD

**Discussion by ricks03** [fudforum/FUDforum#45] [General]
I see that the current version is 3.1.3. downloadable as a .zip file 3,153,092

However, I also see a version 3.1.3 on sourceforge, where the zip file is 8,062,188

That's a fairly big difference (not that there should be any difference at all). What's up with the two files of .php files being 5 MB different?

**Comment by naudefj** [comment]
The version on Github contains additional patches and translations that is destined for FUDforum 3.1.4.
